io/src/lib.rs

```diff
@@ -11,7 +11,7 @@ pub struct InitConfig {
     pub decimals: u8,
 }
 
-#[derive(Debug, Decode, Encode, TypeInfo)]
+#[derive(Debug, Decode, Encode, Copy, Clone, TypeInfo)]
```

Comment:
It is not a too lightweight data structure. Do we really need `Copy`? I'd clone it explicitly when needed.
```suggestion
#[derive(Debug, Decode, Encode, Clone, TypeInfo)]
```

Reply:
Fair enough.